draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -195,8 +212,12 @@ if supported, or TURN relay, if not. As noted in {{IPHandling}}, this may
 result in increased media latency and reduced connectivity/increased cost
 (depending on whether the application chooses to use TURN).
 
-The exact impact of this technique is being researched experimentally and will
-be provided before publication of this document.
+One potential mitigation, as discussed in {#privacy}, is to not conceal
+candidates created from {{RFC4941}} IPv6 addresses. This permits connectivity
+even in large internal networks or where mDNS is disabled.
```

Comment:
Should we try to merge this paragraph with the "Special Handling for IPv6 Addresses" paragraph?

Reply:
This section discusses the limitations, the following section provides a potential solution. But I think the privacy section is growing somewhat large, and it may make sense to have another section after {{principle}} where we discuss refinements of the technique. 

See #76.